CHANGELOG.md

```diff
@@ -147,3 +147,11 @@ See https://github.com/disneystreaming/smithy4s/pull/649
 
 Refinements applied on list/map shapes can now produce parameterised types. This allows, for instance, to have generic
 refinements on `list` shapes that produce `cats.data.NonEmptyList` containing the same types of elements.
+
+# 0.17.2
```

Comment:
mmm sorry to be a pain, but would you mind putting this at the top of the file ? the latest changes in changelogs are usually at the top so that people don't have to scroll down to see what's new.

Reply:
No worries, done :)